Products/ZenModel/Organizer.py

```diff
@@ -355,8 +357,12 @@ def getOrganizer(self, path):
         >>> dmd.Events.Status.getOrganizer('/Events/Status/Snmp')
         <EventClass at /zport/dmd/Events/Status/Snmp>
         """
-        if path.startswith("/"): path = path[1:]
-        return self.getDmdRoot(self.dmdRootName).unrestrictedTraverse(path)
+        path = self.prepId(path)
```

Comment:
This is probably the wrong place to do this. You want to do it down below, before the _getOb call.